### PR TITLE
Boolean to lowercase

### LINKED ISSUE #8: Updating all boolean values to lowercase
Update all boolean value checks to use lowercase.

Example of a change; 
`if [ "$a" == "Check"]; then`
should be changed to
`if [ "$a" == "check"]; then`

### PR DESCRIPTION
**Pull request by LeoDiazL**
Changed variables to lowercase, and any other True/False to lowercase.
Some of them are not a must, but changed them to keep consistency.

### CODE REVIEW

File: bitops.schema.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ cloudformation:
             template-file:
               type: string
               export_env: CFN_TEMPLATE_FILENAME
-              required: True
+              required: true
```
Comment (arm4b) [inline bitops.schema.yaml:42]: This part is handled by the BitOps core and not by the plugin itself.

@PhillypHenning Do you think lowercase `true` in the bitops schema would be parsed properly by the BitOps core?

File: deploy.sh [hunk 2]
```diff
@@ -98,7 +98,7 @@ cd $CLOUDFORMATION_ROOT_OPERATIONS
 # bash $SCRIPTS_DIR/aws/sts.get-caller-identity.sh
 
 # always run cfn template validation first
-if [[ "${CFN_TEMPLATE_VALIDATION}" == "True" ]] || [[ "${CFN_TEMPLATE_VALIDATION}" == "true" ]]; then
+if [[ "${CFN_TEMPLATE_VALIDATION}" == "true" ]] || [[ "${CFN_TEMPLATE_VALIDATION}" == "True" ]]; then
```
Comment (arm4b) [inline deploy.sh:101]: The code does the same as before.
What's the difference here?

**Review by arm4b — CHANGES_REQUESTED** [review]
Thanks for the changes @LeoDiazL and finding this 👍 
Having same formatting everywhere helps.

One question in the comment though.

File: bitops.schema.yaml (hunk 1 above)
Comment (arm4b) [reply bitops.schema.yaml:42]: Answering my own question.

Python's `yaml.parse` used in BitOps core should work ok on this, including `yes`, `no` casting it to bool.
Also interesting from the original YAML spec: https://stackoverflow.com/a/53648738

File: deploy.sh (hunk 2 above)
Comment (LeoDiazL) [reply deploy.sh:101]: Consistency. 
There are two more if's right after that, both of them validates lowercase values first.

**Review by arm4b — APPROVED** [review]

**Pull request merged**